lib/components/narrative/trip-tools.js

```diff
@@ -130,7 +151,9 @@ class PrintButton extends Component {
           className='tool-button'
           onClick={this._onClick}
         >
-          <i className='fa fa-print' /> Print
+          <i className='fa fa-print' />
+          {' '}
```

Comment:
I think adding whitespace is a no-no when using internationalization. If this is for styling purposes to add padding or something, the styling should be changed instead.

Reply:
Are you referring to the layout for right-to-left languages? It looks like `{' '}` is doing the desired effect (see the whitespace at the left of the button text in the example screenshot below).
![image](https://user-images.githubusercontent.com/56846598/130106475-cbe5e50b-3829-4652-8bf1-3f8fa1c5ddd4.png)

Reply:
Otherwise, a more modern way to do it is to use the CSS attribute `margin-inline-end` on the `<i>` element, would that be your recommendation @evansiroky?

Reply:
(Right-to-left support is a whole other topic that reaches way beyond just introducing react-intl.)

Reply:
CSS spacing above applied in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/c9c12725fb9d755721ba93873fbf92dc5115830c, let me know if that's the approach we want to pursue going forward.

Reply:
CSS `margin-line-end` might be too recent of an addition, I used slightly different CSS in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/80ec210071b1236d7818de73273784b000da667c.